package.json

```diff
@@ -9,15 +9,18 @@
     ".": {
       "development": {
         "import": "./lib/es2015/esm/single-spa.dev.js",
-        "require": "./lib/es2015/umd/single-spa.dev.cjs"
+        "require": "./lib/es2015/umd/single-spa.dev.cjs",
+        "types": "./typings/single-spa.d.ts"
```

Comment:
Do we need separate types for import/require? With a `.cts` extension? I'm looking at https://github.com/single-spa/single-spa-react/pull/197/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R16 for reference and am unsure